project/Settings.scala

```diff
@@ -44,6 +46,15 @@ object Settings extends Build {
     versionStatus        := Versions.status(scalaVersion.value, scalaBinaryVersion.value)
   )
 
+  lazy val sparkPackageSettings = Seq(
```

Comment:
I would suggest adding (`++`) this to `settings` on L58 instead of `assembledSettings`